modules/dnn/src/tensorflow/tf_graph_simplifier.cpp

```diff
@@ -682,6 +682,15 @@ void RemoveIdentityOps(tensorflow::GraphDef& net)
             IdentityOpsMap::iterator it = identity_ops.find(input_op_name);
 
             if (it != identity_ops.end()) {
+                // In case of Identity after Identity
```

Comment:
Is this some leftover changes from other patch?

Reply:
Ooops! Fixed.